clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -537,51 +586,137 @@ private void updateGroupSubscription(Set<String> topics) {
         return groupAssignment;
     }
 
+    /**
+     * Used by COOPERATIVE rebalance protocol only.
+     *
+     * Validate the user assignor returned assignments such that there are no partitions which are going to
+     * be assigned to a different consumer other than its current owner (i.e. who includes it in its owned
+     * partitions list encoded in subscription); also if there are any revoked partitions set the error code to let
+     * everyone retry.
```

Comment:
Need to update the docs since error code was removed. Also, maybe "Validate the user assignor returned assignments" -> "Validate the assignments returned by the assignor" ?

Reply:
Ack.